airflow/configuration.py

```diff
@@ -1881,7 +1880,7 @@ def get_airflow_config(airflow_home: str) -> str:
 
 
 def get_all_expansion_variables() -> dict[str, Any]:
-    return {k: v for d in [globals(), locals()] for k, v in d.items()}
+    return {k: v for d in [globals(), locals()] for k, v in d.items() if not k.startswith("_")}
```

Comment:
Not _required_, but this made debugging easier -  this excludes `__doc__` and `__builtins__` etc.